### PR TITLE
Zo'ra and K'lax Bound for ops are now from Scay and Vedhra

### PR DESCRIPTION
**Pull request by Sniblet**
Previously, their colors indicated Zoleth (why) and Vetju (_big_ why). Now, each hive is represented by the Queen most likely to actually invest their Bound in the SCC's research. I didn't change Vytel because she still makes the most sense for C'thur.
Have a bug report while I'm here: sometimes, the bound workers that ops gets are named Za and/or Akaix, because every bug uses the same random-naming scheme regardless of type and origin. And I don't know how to fix this.

### CODE REVIEW

**Comment by Sniblet** [comment]
!review

File: code/modules/mob/living/carbon/human/human_species.dm [hunk 1]
```diff
@@ -74,11 +74,11 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy/mannequin)
 	var/list/hive = splittext(name, " ")
 	switch(hive[length(hive)])
 		if("K'lax")
-			change_skin_color(33, 63, 33)
+			change_skin_color(20, 170, 20) //vedhra does bio-research
```
Comment (SleepyGemmy) [inline code/modules/mob/living/carbon/human/human_species.dm:77]:
```suggestion
			change_skin_color(20, 170, 20) // Vedhra does bioresearch.
```

File: code/modules/mob/living/carbon/human/human_species.dm [hunk 2]
```diff
@@ -74,11 +74,11 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy/mannequin)
...
 		if("C'thur")
-			change_skin_color(10, 35, 55)
+			change_skin_color(10, 35, 55) //vytel tolerates the scc the most
```
Comment (SleepyGemmy) [inline code/modules/mob/living/carbon/human/human_species.dm:79]:
```suggestion
			change_skin_color(10, 35, 55) // Vytel tolerates the SCC the most.
```

File: code/modules/mob/living/carbon/human/human_species.dm [hunk 3]
```diff
@@ -74,11 +74,11 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy/mannequin)
...
 		if("Zo'ra")
-			change_skin_color(111, 21, 21)
+			change_skin_color(71 ,11, 51) //scay does bio-research
```
Comment (SleepyGemmy) [inline code/modules/mob/living/carbon/human/human_species.dm:81]:
```suggestion
			change_skin_color(71 ,11, 51) // Scay does bioresearch.
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**